firestore-bigquery-export/functions/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "target": "ES2020",
     "module": "commonjs",
     "noImplicitReturns": true,
-    "sourceMap": false
+    "sourceMap": false,
+    "skipLibCheck": true
```

Comment:
What does this do? (Why do we need it?)

Reply:
Ended up being caused by the out of date @types/node package. Removed.